### DISCUSSION TITLE
define Master Password length

### DISCUSSION THREAD

**Discussion by Muzafferhashemi** [keepassxreboot/keepassxc#6547] [Q&A]
Hi, 

is there a possibility to define the Master Password length or generally set requirements beforehand, like in the configuration .ini file?
Also it should popup and error if the Password Requirements are not fulfilled.

Thanks.

**Comment by Muzafferhashemi** [comment]
sorry, i forgot to mention, it's for MacOS.

**Comment by droidmonkey** [comment]
No, there is not. This requirement doesn't really make sense unless we have enforced administrate settings.

**Reply by Muzafferhashemi** [reply]
Thats exactly what we want to do. We have Jamf Pro and and we want to enroll the Application with predefined settings. We want to make sure that the User is using a strong Master Password. Is there a possibility to realize this option?

**Comment by simsonntag** [comment]
Hello,

I ask myself the same question. Can I force a minimum master password complexity within the keepassxc.ini via deployment?
